src/client/Microsoft.Identity.Client/Internal/AndroidBrokerTokenResponse.cs

```diff
@@ -1,74 +1,74 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
Spacing... 
No change made to this file, only a rename to AndroidBrokerResponse.